### PR TITLE
auth , cartmanagement and wishlist added

### PR DESCRIPTION
**Pull request by MiryalaAkshitha**

### CODE REVIEW

**Review by vivekgudipudi — APPROVED** [review]
clean UI, ready to merge

File: src/App.js [hunk 1]
```diff
@@ -2,6 +2,13 @@ import "./App.css";
 import { Homepage } from "./pages/Homepage.js";
 import {ProductListing} from "./pages/Product-Listing.js";
 import {Routes,Route} from "react-router-dom";
+import { Cart } from "./pages/CartPage.js";
+import { Wishlist} from "./pages/WishListPage.js"
+import Login from "./components/Auth/Login";
+import Signup from "./components/Auth/Signup.js";
+import NotFound from "./pages/NotFound.js";
+import MockmanEs from "mockman-js";
```
Comment (MaruthiPothuganti) [inline src/App.js:10]: Suggestion: Try barrel exports from pages

**Review by shashikantmaurya0007 — APPROVED** [review]
code looks good to be merged

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a change. Everything else looks good.

**Pull request merged**